CONTRIBUTING.md

```diff
@@ -42,8 +42,14 @@ cloudfour.com-patterns
 1. `git checkout v-next`
 2. `git pull`
 3. Make sure you have a clean working tree (`git status` should show no changes)
-4. `npm version` - This will bump the version number in `package.json` and `package-lock.json`
-5. `npm publish --access public` - This will automatically install and compile everything, run linting, and publish
+4. `git checkout release-X.Y.Z` Where `X.Y.Z` is the version number you're about to release.
```

Comment:
Is this missing `-b`? Or does `git checkout` automatically create a branch now if it doesn't exist?

Reply:
Nope you're right!